streams/ci.m3u

```diff
@@ -1,3 +1,7 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="NTV.ci" status="online",NTV Afrique (1080p) [Not 24/7]
 https://strhlslb01.streamakaci.tv/str_ntv_ntv/str_ntv_ntv_multi/playlist.m3u8
+#EXTINF:-1 tvg-id="IvoireChannel.ci" status="",Ivoire Channel (720p)
```

Comment:
Shares its video flux with Fixion Channel right now.

Reply:
?

Reply:
There's two channels streaming in the same link : Ivoire channel in the morning + afternoon, Fixion Channel in the evening.